--- qt-everywhere-src-6.10.1/qtwebengine/src/3rdparty/chromium/sandbox/linux/system_headers/linux_seccomp.h.orig	2025-11-14 08:55:10.000000000 +0100
+++ qt-everywhere-src-6.10.1/qtwebengine/src/3rdparty/chromium/sandbox/linux/system_headers/linux_seccomp.h	2026-01-29 22:52:59.857202173 +0100
@@ -5,6 +5,7 @@
 #ifndef SANDBOX_LINUX_SYSTEM_HEADERS_LINUX_SECCOMP_H_
 #define SANDBOX_LINUX_SYSTEM_HEADERS_LINUX_SECCOMP_H_
 
+#include <features.h>
 #include <stdint.h>
 #include <sys/ioctl.h>
 
@@ -214,8 +215,10 @@
 #define SECCOMP_RET_INVALID 0x00010000U  // Illegal return value
 #endif
 
+#if !defined(__GLIBC__) || !defined(__GLIBC_MINOR__) ||  __GLIBC__ < 2 || (__GLIBC__ == 2 && __GLIBC_MINOR__ < 43)
 #ifndef SYS_SECCOMP
 #define SYS_SECCOMP                   1
 #endif
+#endif
 
 #endif  // SANDBOX_LINUX_SYSTEM_HEADERS_LINUX_SECCOMP_H_
