--- qt-everywhere-src-6.3.1/qtwebengine/src/CMakeLists.txt.orig	2022-06-08 08:40:31.000000000 +0200
+++ qt-everywhere-src-6.3.1/qtwebengine/src/CMakeLists.txt	2022-07-27 21:47:18.766238322 +0200
@@ -137,6 +137,8 @@
                    -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>
                    -DCMAKE_PREFIX_PATH:PATH=<INSTALL_DIR>
                    -DWEBENGINE_ROOT_BUILD_DIR=${PROJECT_BINARY_DIR}
+                   -DNinja_EXECUTABLE:FILEPATH=${Ninja_EXECUTABLE}
+                   -DCMAKE_MAKE_PROGRAM:FILEPATH=${CMAKE_MAKE_PROGRAM}
     )
     if(QT_FEATURE_qtwebengine_core_build)
         add_dependencies(run_core_GnReady gn)
