--- qt-everywhere-src-6.10.3/qtwebengine/cmake/Functions.cmake.orig	2026-03-30 14:29:09.000000000 +0200
+++ qt-everywhere-src-6.10.3/qtwebengine/cmake/Functions.cmake	2026-04-02 12:03:21.328240391 +0200
@@ -654,6 +654,8 @@
                    -DCMAKE_VERBOSE_MAKEFILE=${CMAKE_VERBOSE_MAKEFILE}
                    -DQT_ALLOW_SYMLINK_IN_PATHS=${QT_ALLOW_SYMLINK_IN_PATHS}
                    -DPython3_EXECUTABLE=${Python3_EXECUTABLE}
+                   -DNinja_EXECUTABLE:FILEPATH=${Ninja_EXECUTABLE}
+                   -DCMAKE_MAKE_PROGRAM:FILEPATH=${CMAKE_MAKE_PROGRAM}
                    -DGCC_LEGACY_SUPPORT=${QT_FEATURE_webengine_gcc_legacy_support}
     )
     unset(OSX_ARCH_STR)
