--- qt-everywhere-src-6.3.1/qtwebengine/src/gn/CMakeLists.txt.orig	2022-06-08 08:40:31.000000000 +0200
+++ qt-everywhere-src-6.3.1/qtwebengine/src/gn/CMakeLists.txt	2022-07-29 16:53:34.712173434 +0200
@@ -32,6 +32,7 @@ add_custom_command(
     WORKING_DIRECTORY ${GN_BINARY_DIR}
     COMMAND ${Python3_EXECUTABLE} ${GN_SOURCE_DIR}/build/gen.py
         --no-last-commit-position
+	--no-static-libstdc
         --out-path ${GN_BINARY_DIR}/$<CONFIG>
         --cc ${CMAKE_C_COMPILER}
         --cxx ${CMAKE_CXX_COMPILER}
